upup/pkg/fi/cloudup/new_cluster.go

```diff
@@ -969,12 +969,13 @@ func setupTopology(opt *NewClusterOptions, cluster *api.Cluster, allZones sets.S
 
 		if opt.IPv6 {
 			cluster.Spec.NonMasqueradeCIDR = "::/0"
-			cluster.Spec.ExternalCloudControllerManager = &api.CloudControllerManagerConfig{}
+			cluster.Spec.ExternalCloudControllerManager = &api.CloudControllerManagerConfig{
+				CloudProvider: "aws",
```

Comment:
Can we do this through defaulting the value downstream?

Reply:
We could just replace `eccm.CloudProvider` with `cluster.Spec.CloudProvider` here:
https://github.com/kubernetes/kops/blob/3fd7b446c06a04d9c133bb37c63c12b3ae560415/pkg/model/components/awscloudcontrollermanager.go#L41-L43

Reply:
Does it look better?